app/components/external-link.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I see that this is the file that's being pulled into the component, rather than the one in app/svgr.
Please remove this file, and switch to importing the react component from app/svgr
and use the name SvgExternalLink as a convention - so that it's more apparent that it's an Svg.  You may have to so some styling to get the size needed.